Docuement change on teamSlugs size on cards filter (#556)
sorare/api: CHANGELOG.md

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).

## 2023-12-12

> limit the number of allawed `teamSlugs` in card''s field arguments. impacted fields: `allCards`, `cards, paginatedCards`, `footballCards`, `football.cards`, `football.allCards`, `team.cards`, `player.cards`

Reviewer: ```suggestion
Limit the number of allowed `teamSlugs` on the `cards` field arguments. Impacted fields include: `allCards`, `cards`, `paginatedCards`, `footballCards`, `football.cards`, `football.allCards`, `team.cards`, `player.cards`.
```
